services-camel/src/main/webapp/WEB-INF/solr-indexing-context.xml

```diff
@@ -268,10 +268,11 @@
     <bean id="indexTreeInplaceAction"
         class="edu.unc.lib.dl.data.ingest.solr.action.IndexTreeInplaceAction">
         <property name="pipeline" ref="solrFullUpdatePipeline" />
-        <property name="solrSettings" ref="solrSettings" />
         <property name="solrSearchService" ref="queryLayer" />
         <property name="accessGroups" ref="accessGroups" />
         <property name="updateDelay" value="${conductor.solr.beforeExecuteDelay}"/>
+        <property name="repositoryObjectLoader" ref="repositoryObjectLoader" />
```

Comment:
what do these services-camel changes have to do with the other changes on this PR?